samples/examples/image_classifier/Makefile

```diff
@@ -0,0 +1,18 @@
+VERSION := $(shell grep seldonio/mlserver ../../../scheduler/Makefile | cut -d':' -f2)
```

Comment:
should we move this under `samples/scripts/` for consistency?

Reply:
maybe in a future PR - as need to discuss how these more complex examples are trained together